package.json

```diff
@@ -93,7 +93,8 @@
     "rules": {
       "import/prefer-default-export": 0,
       "react/no-unused-prop-types": 0,
-      "valid-jsdoc": 0
+      "valid-jsdoc": 0,
+      "func-style": 0
```

Comment:
I got annoyed by this rule. I still prefer function declarations, but I don't care to enforce it anymore.